.travis.yml

```diff
@@ -24,7 +25,7 @@ cache:
 
 script:
   # need to override as the default is to test
-  - sbt -J-XX:ReservedCodeCacheSize=256m +update
+  - sbt -J-XX:ReservedCodeCacheSize=256m update mimaReportBinaryIssues
```

Comment:
@2m how can I make sbt do `++{the-scalaversion}`? I want it to be done in parallel, which is why the setting above and not just + here

Reply:
Do `++$TRAVIS_SCALA_VERSION`